design/token.go

```diff
@@ -100,6 +100,7 @@ var _ = a.Resource("token", func() {
 	})
 
 	a.Action("Exchange", func() {
+		a.Security("jwt")
```

Comment:
It will break API. Not all exchange call require JWT security / expect access token in Authorization header.